RUFAS/routines/animal/pen.py

```diff
@@ -407,9 +407,29 @@ def update_animals(self, new_animals: List[Any],
         self.update_animal_combination(animal_combination)
         self.update_classes_in_pen()
 
+    def manure_sums(self, manure, curr_manure, dict):
+        """
+        Accumulator helper function for calc_manure.
+        The function finds sums of manure components for each
+        animal in the pen and the total manure for each animal type.
+        Parameters
+        ----------
+            manure: Dict[float, int]
+                A dictionary that contains the the accumulated maniure excretion values for all animals
+            curr_manure: AnimalManureExcretions
+                A dictionary that contains the manure excretion values as specified
+                in the AnimalManureExcretions class definition.
+            dict: Dict[float, int]
+                A dictionary that contains the manure excretion values for specific animals in the pen
+        """
```

Comment:
```suggestion
                A dictionary that contains the manure excretion values for specific animals in the pen

        """
```
There should be an extra line between the last line of docstrings and the closing """